posts/2024-06-11-python-tutorial-part-2/2024-06-11-python-tutorial-part-2.qmd

```diff
@@ -0,0 +1,12 @@
+---
+title: "Python tutorial - Part 2"
+author: "Camille Coux"
+date: "2024-06-11"
+categories: [python, tutorial, games]
```

Comment:
Can you please replace games by one or more different topics?

Reply:
Solved in [e4d1df9](https://github.com/FRBCesab/tips-and-tricks/pull/28/commits/e4d1df933738c6ad99b47a8c77989bf6b7c9a9c4)